src/class/Cars.js

```diff
@@ -0,0 +1,60 @@
+import Car from "./Car";
+
+const CAR_ADVANCE_MAX_NUMBER = 9;
+
+const CAR_ADVANCE_THRESHOLD_NUMBER = 4;
+
+const DEFAULT_RACING_ROUND_NUMBER = 5;
+
+const defaultAdvanceCondition = () => {
+  return Math.random() * CAR_ADVANCE_MAX_NUMBER >= CAR_ADVANCE_THRESHOLD_NUMBER;
+};
+
+export default class Cars {
+  cars;
+  advanceCondition;
+  roundNumber;
```

Comment:
```suggestion
const defaultAdvanceCondition = () => {
  return Math.random() * CAR_ADVANCE_MAX_NUMBER >= CAR_ADVANCE_THRESHOLD_NUMBER;
};

export default class Cars {
  #CAR_ADVANCE_MAX_NUMBER = 9;
  #CAR_ADVANCE_THRESHOLD_NUMBER = 4;
  #DEFAULT_RACING_ROUND_NUMBER = 5;

  cars;
  advanceCondition;
  roundNumber;
```

이렇게 Cars와 관련된 상수는 Cars 내부에 선언하면 되지 않을까 싶기도 하네요!